src/test/skript/tests/misc/default variables.sk

```diff
@@ -0,0 +1,28 @@
+
+variables:
+	{testing variables1::%entity%} = 1337
+	{testing variables2::%object%} = 1337
+	{testing variables3::%entity%::%object%} = 1337
+
+on test "default variables":
+	spawn a pig at spawn of world "world"
+	assert {testing variables1::%last spawned pig%} is set with "default variable 1 was not set"
```

Comment:
Remove redundant checks (if it's unset then it won't be 1337)

Reply:
What

Reply:
I think kiip means the `is set` check is redundant since it is covered by the subsequent `is 1337` check.

Reply:
I disagree since the main focus is to check against default variables.

Reply:
The only situation where this check would be ok is when you're testing the equal condition. In this test we may expect conditions to work correctly, so the check `is set` is redundant when there also is a `is 1337` check.